### DISCUSSION TITLE
Are there any advanced math symbols keyboards?

### DISCUSSION THREAD

**Discussion by capsicumw** [AnySoftKeyboard/AnySoftKeyboard#3853] [Q&A]
Are there any dedicated keyboards for the "language" of advanced math and logic symbols? Like some reasonable selection from the unicode blocks listed at https://en.wikipedia.org/wiki/Mathematical_operators_and_symbols_in_Unicode
It's sort of a language of its own but I haven't found any such add-ons. (Of course a user would need a typeface that covers these sections of unicode.)

**Comment by homchom** [comment]
I discovered this project for exactly such a keyboard. I'm thinking I might make an add-on myself, but I'll need to figure out the specifics of how.
